nixos/modules/services/desktops/playerctld.nix

```diff
@@ -0,0 +1,30 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.playerctld;
+in
+{
+  options.services.playerctld = {
+    enable = mkEnableOption (lib.mdDoc "the playerctld daemon");
+
+    package = mkPackageOption pkgs "playerctl" { };
+  };
+
+  config = mkIf cfg.enable {
+    environment.systemPackages = [ cfg.package ];
```

Comment:
The package does not need to be installed globally.

Reply:
fixed

Reply:
Sorry, I take this back. It's done everywhere, and in this case, the daemon can also be controlled by subcommands.